server/cron/calendar_cron.go

```diff
@@ -391,6 +394,29 @@ func processFailingHostExistingCalendarEvent(
 	updated := false
 	now := time.Now()
 
+	// Function to generate calendar event body.
+	var generatedTag string
+	genBodyFn := func(conflict bool) (string, bool, error) {
```

Comment:
Why do we define these ad-hoc instead of having a `genBodyFn` type somewhere?

Reply:
That's a good idea. I haven't thought of that. I'll make the change.